inc/osvr/Util/EyeTrackerReportTypesC.h

```diff
@@ -0,0 +1,71 @@
+/** @file
+@brief Header
+
+Must be c-safe!
+
+@date 2015
+
+@author
+Sensics, Inc.
+<http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_EyeTrackerReportTypesC_h_GUID_7BD4C450_5D26_4090_BCA4_7305E3DCC6B7
+#define INCLUDED_EyeTrackerReportTypesC_h_GUID_7BD4C450_5D26_4090_BCA4_7305E3DCC6B7
+
+
+// Internal Includes
+#include <osvr/Util/Vec2C.h>
+#include <osvr/Util/Vec3C.h>
+#include <osvr/Util/ChannelCountC.h>
+
+// Library/third-party includes
+// - none
+
+// Standard includes
+// - none
+
+OSVR_EXTERN_C_BEGIN
+
+/** @brief Type for buffer access to eye data */
+typedef unsigned char OSVR_EyeBufferElement;
+
+typedef enum OSVR_Eye_Type {
```

Comment:
Do these two enums need to be here? I figured this data would be in the JSON descriptor.